doc/man_docs.go

```diff
@@ -65,6 +65,12 @@ func GenManTreeFromOpts(cmd *cobra.Command, opts GenManTreeOptions) error {
 	if opts.CommandSeparator != "" {
 		separator = opts.CommandSeparator
 	}
+
+	err := os.Mkdir(opts.Path, 0766)
```

Comment:
I'm not entirely convinced I like the idea at all. I'm trying to think of over unix commands which make a non-existing directory. This means that a typo will go unnoticed and generate docs where the user won't look for them. Before it would have said that the dir didn't exist. The use could either fix their typo or create the dir....